src/styles/unils/_variables.scss

```diff
@@ -0,0 +1,18 @@
+$cal-width: 100px;
+$fontSize: 30px;
+$cal-gap: 1px;
+
+$background-day-color: #eee;
+$background-day-color-hover: #898686;
+$background-day-color-border-hover: #1e1818;
+
+$days:
+  "mon" 0,
+  "tue" 1,
+  "wed" 2,
+  "thu" 3,
+  "fri" 4,
+  "sat" 5,
+  "sun" 6;
+
+
```

Comment:
extra empty lines
```suggestion
```

Reply:
Don't use more than one empty line at the end